gulpfile.js

```diff
@@ -86,7 +86,7 @@ function write(streams, name) {
   return concat(function(body) {
     return file(dest, body, {src: true})
       .pipe(rev())
-      .pipe(gulpif(production, uglify()))
+      .pipe(uglify())
```

Comment:
Hm, good point. I just checked and I think it's a case of the production variable not working correctly if prod isn't being uglified.